docs/src/gcode/o-code_fr.adoc

```diff
@@ -183,18 +311,37 @@ O102 endif
 
 .Exemple avec 'if - elseif - else - endif'
 ----
-o102 if [#2 GT 5] (si le paramètre #2 est supérieur à 5 alors F100)
+(if parameter #2 is greater than 5 set F100)
+O102 if [#2 GT 5]
+  F100
+(else if parameter #2 less than 2 set F200)
+O102 elseif [#2 LT 2]
+  F20
+(parameter #2 is between 2 and 5)
+O102 else
+  F200
+O102 endif
+----
+
+Several conditions may be tested for by 'elseif' statements until the
+'else' path is finally executed if all preceding conditions are false:
+
+.If Elseif Else Endif Example
+----
+(if parameter #2 is greater than 5 set F100)
+O102 if [#2 GT 5]
   F100
-o102 elseif [#2 LT 2] (sinon si le paramètre #2 est inférieur à 2 alors F200)
+(else if parameter #2 less than 2 set F200)
```

Comment:
```suggestion
(sinon si le paramètre #2 est inférieur à 2 alors F200)
```

Reply:
This is a bit of code that should not be translated?

Reply:
Understand that, but comments in this chunk are really part of the explanation of this section.